docs/tutorials/google/spin_echoes.ipynb

```diff
@@ -0,0 +1,1362 @@
+{
```

Comment:
<p>Moment 2 has single qubit gate (<code>PhX(-0.75)^0.5</code>) and two qubit gate (<code>sqrt(iswap)</code>)</p><p> </p><p>Note that categories should be an <code>Iterable[Category]</code> and <code>Classifier = Callable[['cirq.Operation'], bool]</code> , so <code>categories = [lambda op : len(op.qubits) == 1, lambda op : len(op.qubits) == 2]</code> should do the intended stratification. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/quantumlib/Cirq/pull/4373/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='40'/>

Reply:
<p>Fixed, thanks!</p>